packages/firestore/test/lite/integration.test.ts

```diff
@@ -2047,8 +2047,8 @@ describe('withConverter() support', () => {
 
 describe('countQuery()', () => {
   it('AggregateQuery and AggregateQuerySnapshot inherits the original query', () => {
-    return withTestCollection(async coll => {
-      const query_ = query(coll);
+    return withTestCollection(async collection => {
```

Comment:
Avoid naming variables `collection` because that is also the name of a top-level function in the Firestore SDK. That's why `coll` gets used a lot.

Reply:
I see 🤔